newsroom/agenda/agenda.py

```diff
@@ -0,0 +1,102 @@
+
+import newsroom
+from planning.events.events_schema import events_schema
+from planning.planning.planning import planning_schema
+from superdesk.metadata.item import not_analyzed
+from planning.common import WORKFLOW_STATE_SCHEMA
+from newsroom.wire.search import get_local_date
+from eve.utils import ParsedRequest
+from flask import json
+
+
+class AgendaResource(newsroom.Resource):
+    """
+    Agenda schema
+    """
+    schema = {}
+
+    # identifiers
+    schema['event_id'] = events_schema['guid']
+    schema['planning_id'] = planning_schema['guid']
+
+    # content metadata
+    schema['name'] = events_schema['name']
+    schema['slugline'] = events_schema['slugline']
+    schema['definition_long'] = events_schema['definition_long']
+    schema['abstract'] = planning_schema['abstract']
+    schema['headline'] = planning_schema['headline']
+    schema['firstcreated'] = events_schema['firstcreated']
+    schema['versioncreated'] = events_schema['versioncreated']
+
+    # normalized dates
+    schema['dates'] = {
+        'type': 'list',
+        'nullable': True,
+        'schema': {
+            'type': 'dict',
+            'schema': {
+                'start': {'type': 'datetime'},
+                'end': {'type': 'datetime'},
+                'state': WORKFLOW_STATE_SCHEMA,
+                'planning_id': planning_schema['guid'],
+                'coverages': {'type': 'list', 'nullable': True},
+            }
+        },
+        'mapping': not_analyzed,
```

Comment:
I think for the mapping we will need a [nested objects] in order to filter by start/end